tools/torch_quant/tests/test_quantizer.py

```diff
@@ -21,7 +21,7 @@
 class QuantizerTest(unittest.TestCase):
     @parameterized.expand([
         (Backend.REFERENCE, ),
-        (Backend.FBGEMM, ),
+        # (Backend.FBGEMM, ), TODO(litan.ls): select test according to ci env
```

Comment:
Why do we choose what to test based on environment variables?

Reply:
some environments (dev images) don't have specific backend. For example, aarch64 env doesn't have FBGEMM.